en/creating-artboard-adobexd.md

```diff
@@ -0,0 +1,216 @@
+    2.  Place them one under the other with a vertical spacing of 16px between each other and to the ones from the previous step.
+    3.  Stretch them across the full width with 16px spacing on both sides.
+    4.  Set their Labels to "Username" and "Password" and their Values to "Leaellynasaura" and "\*\*\*\*\*\*\*\*\*\*\*\*".
+    5.  Select the hidden ContentType layer of the Password input and change its state to Password.
+
+    <div class="divider--half"></div>
+    <img class="responsive-img" src="images/getting-startedxd5.png" srcset="images/getting-startedxd5@2x.png 2x" />
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+
+7.  **Add the sign up button.**
+
+    1.  Change the left panel to **Libraries** and look for a Raised Button.
+    2.  Drag it below the inputs, making sure you pick the right component, not the one prefixed with _Overrides, and stretch it across the full width leaving 16px on the left, right, and from the inputs above.
+    3.  Switching back to **Layers**, expand the component, delete its icon and set the Label to "SIGN UP".
+    4.  Select the Group wrapping the Label and position it in the middle of the Button.
+    5.  Switch again to **Libraries** to give the label a surface color and set the Background to info color.
+
+    <div class="divider--half"></div>
+    <img class="responsive-img" src="images/getting-startedxd6.png" srcset="images/getting-startedxd6@2x.png 2x" />
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+
+8.  **Add the terms acceptance text.**
+
+    1.  Press the `T` key to add another text element below the button and type "By clicking on the "SIGN UP" button above, you accept our Terms of Use" in it.
+    2.  Assign a Caption - 12pt Character Style to it and give it a grays.700 Color.
+    3.  From the right panel set the Text to Auto Height and change the layer size so that there is a 16px spacing on the left, right and above.
+
+    <div class="divider--half"></div>
+    <img class="responsive-img" src="images/getting-startedxd7.png" srcset="images/getting-startedxd7@2x.png 2x" />
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+    <div class="divider--half"></div>
+
+9.  **Make the artboard responsive.**
+
+Select the text layers, inputs, button and white rectangle, and group them. The automatic Responsive Layout for this group is enough to assure the expected responsive design behavior. 
+
+## Additional Resources
+
+Related topics:
+
+- [Creating Your First Artboard in Sketch](creating-artboard-sketch.md)
+
```

Comment:
Double check the spacing here, it looks weird
<img width="480" alt="Screen Shot 2021-07-22 at 3 43 50 PM" src="https://user-images.githubusercontent.com/41998471/126641041-b2255290-bf53-4c68-8286-bc3bd02ff4c2.png">